docs/Users_Guide/roc_diagram.rst

```diff
@@ -51,27 +52,103 @@ METplus Configuration
 **Default Configuration File**
 
 The following is the `mandatory`, roc_diagram_defaults.yaml configuration file,
-which serves as a good starting point for creating a ROC diagram
-plot as it represents the default values set in METviewer
+which serves as a starting point for creating a ROC diagram plot,  as it represents
+the default values set in METviewer.
+
+**NOTE**: This default configuration file is automatically loaded by roc_diagram.py.
 
 .. literalinclude:: ../../metplotpy/plots/config/roc_diagram_defaults.yaml
 
+
+
+Modify the `stat_input` setting (input data file) in this default config file to ensure that the data file is accessible.
+Open the $METPLOTPY_SOURCE/METplotpy/metplotpy/plots/config/roc_diagram_defaults.yaml file and replace
+the relative path in the stat_input entry `stat_input:  ../../../test/roc_diagram/plot_20200507_074426.data`
+with the path to your $METPLOTPY_SOURCE directory.  For example, if your $METPLOTPY_SOURCE is /username/plotting,
+then the `stat_input` setting will look like the following:
+
+`stat_input:  /username/plotting/METplotpy/test/roc_diagram/plot_20200507_074426.data`
+
+
 **Custom Configuration File**
 
+
 A second, `mandatory` configuration file is required, which is
-used to customize the settings to the ROC diagram plot. The custom_reliability.yaml
+used to customize the settings to the ROC diagram plot. The custom_roc_diagram.yaml
 file is included with the source code and looks like the following:
 
-.. literalinclude:: ../../metplotpy/plots/roc_diagram/custom_roc_diagram.yaml
+.. literalinclude:: ../../test/roc_diagram/custom_roc_diagram.yaml
+
+copy this custom config file from the directory where you saved the source code to your working directory:
+
+``cp $METPLOTPY_SOURCE/METplotpy/test/roc_diagram/custom_roc_diagram.yaml $WORKING_DIR/custom_roc_diagram.yaml``
+
+
+Modify the `stat_input` and `plot_filename` settings in the
+$METPLOTPY_SOURCE/METplotpy/test/roc_diagram/custom_roc_diagram.yaml
+file to explicitly point to the $METPLOTPY_SOURCE/METplotpy/test/roc_diagram/roc_diagram directory (where
+the custom config files and sample data reside).  Replace the relative path `./plot_20200507_074426.data`
+with the full path `$WORKING_DIR/METplotpy/test/roc_diagram/plot_20200507_074426.data`
+
+For example:
+
+`stat_input: /username/working_dir/METplotpy/test/roc_diagram/plot_20200507_074426.data`
+
```

Comment:
Users may also wish to change the path listed in the value of `plot_filename` to an output directory of their choice.  Otherwise, the plot output will be saved in the current working directory.  For example: